botocore/httpsession.py

```diff
@@ -45,7 +45,10 @@ def get_cert_path(verify):
     if verify is not True:
         return verify
 
-    return where()
+    cert_path = where()
+    logger.debug("Certificate path: {0}".format(cert_path))
```

Comment:
Just as a stylistic note, we don't need to enumerate formatting args anymore and can just use `{}` or `{cert_path}`. The numbering requirement was dropped in Python 2.6.